### ISSUE TITLE
0.5.0 removes ability to talk/respond in chat

### ISSUE THREAD

**Issue by RoboMWM** [kevinthedang/discord-ollama#65]
Hi, I had a couple questions, not sure if there's a forum to ask about these, but I split them up into two different tickets since I know I'd prefer that for my projects.

The latest release prevents the bot from talking in a channel. I'm assuming this is known as per https://github.com/kevinthedang/discord-ollama/issues/13#issuecomment-2053856406 but I didn't know this until after searching a while, I browsed closed issues in this project. The error indicates that it still is looking for a channel. I've reverted to 0.4.4 for now as I prefer chats instead of threads.

**Comment by kevinthedang** [comment]
Ah shoot! Can you provide any screenshots of any problems?

Are you utilizing docker and having issues?

Are threads kind of unwieldy and unnecessary?

Please let us know what you think! :) we're kind of doing this for fun.

**Comment by kevinthedang** [comment]
@RoboMWM We can also add a command that toggles a thread/channel chatting. Do you think that would be useful? Or would you think users would just prefer chats in the channel?

**Comment by RoboMWM** [comment]
It just says something like "the channel/thread does not exist." I can't recall the exact message unfortunately.

A toggle or config option would be fine. I'm also just playing around with it too. Personally I'd like something that replicates what Discord's "clyde" bot does, which has been only available in very small, active, private servers. Something that can be chatted with like another user in the server, that can use context from other chat messages in the channel and such. This has been the closest I've found to that so far (tho I only found this cuz of lollms, another hobby project, where I discovered ollama, and made me curious if there's other projects using this that work directly with discord lol).
